### ISSUE TITLE
Enhancement: Force recheck of torrents

### ISSUE THREAD

**Issue by iconoclasthero** [teal77/trireme#68]
I actually do this a fair amount from the GTK client.

**Comment by iconoclasthero** [comment]
Huh, I needed this again today for a torrent stuck at 100% but not "completed." I went to put in a ticket to ask for it, but I see someone has already put one in: me.

**Comment by iconoclasthero** [comment]
Oh, also, there's no info page anywhere including the hamburger menu/settings to show this git and the current-installed version (1.3).

**Comment by teal77** [comment]
Hi, This functionality should already be there as one of the torrent details bottom toolbar buttons. Although I suppose the icon could be clearer.

**Comment by iconoclasthero** [comment]
You sure about that?  I'm looking for Force Re-check

[image: Screenshot from 2023-04-10 22-20-27.png]
not update tracker which is what the two arrows is:

[image: 0048a28f88316b36527a2f15831c5b48d4b9fe1c-1.jpeg]
...

**Comment by iconoclasthero** [comment]
Bummer you can't email in the screenshots.  :(

**Comment by teal77** [comment]
It should be the search icon, I don't have the app installed at the moment

**Comment by iconoclasthero** [comment]
Yeah, it seems to be there under the magnifying glass icon... That's what
it says it does anyway.  As such, I concur that the icon is lacking but I'm
not sure what would be better.

Please excuse the brevity of this message; it was sent from a mobile device.
